WINDOWS_LINUX_SUBSYSTEM.md

```diff
@@ -238,54 +217,35 @@ stt
 
 **Wait 1 minute** for additional packages to be automatically installed (New tabs with text will automatically open, containing documentation for each new package installed). TO follow package installation, you can go to `View > Show console`.
 
-To check if plugins are installed, open the Command Palette (`⌘` + `⇧` + `P` on OSX, `Ctrl` + `⇧` + `P` on Linux), type in `Packlist` and then `Enter`, you should see a couple of packages installed (like [Emmet](http://emmet.io/)).
+To check if plugins are installed, open the Command Palette with `Ctrl` + `⇧` + `P`, type in `Packlist` and `Enter`, and you should see a couple of packages installed (like [Emmet](http://emmet.io/)).
 
 If you don't, please install all of them manually. The list is referenced [here](https://github.com/lewagon/dotfiles/blob/master/Package%20Control.sublime-settings).
 
 When it's done, you can close Sublime Text.
 
-### Auto-start `ssh-agent`
```

Comment:
How do you deal with it?

Reply:
I'd go with passwordless key files.

I removed this part here, because every student who attempted this, got it wrong. And unless we figure out a better way to explain this, I would go with passwordless key files.

Reply:
this works like a charm for me